mapmatcher/trip.py

```diff
@@ -275,6 +287,14 @@ def __pre_process(self):
             if len(self._waypoints.stop_node.unique()) < 2:
                 self._err.append("All valid GPS ping map to a single point in the network")
 
+        if outside.shape[0] > 0:
```

Comment:
Should we allow this to be turned off?
```suggestion
        if save_ping_classification && outside.shape[0] > 0:
```